pkg/service/schema/model.go

```diff
@@ -49,9 +37,9 @@ type GetSchemaRequest struct {
 }
 
 type GetSchemaResponse struct {
-	ID        string              `json:"id"`
-	Schema    schema.VCJSONSchema `json:"schema"`
-	SchemaJWT *keyaccess.JWT      `json:"schemaJwt,omitempty"`
+	ID               string            `json:"id"`
+	Schema           schema.JSONSchema `json:"schema"`
+	CredentialSchema *keyaccess.JWT    `json:"credentialSchema,omitempty"`
```

Comment:
Same as above. Suggestion is that `schema.Service` only stores plain schemas.

Reply:
let's take this in the next issue